templates/base.html.twig

```diff
@@ -1,13 +1,5 @@
 <!DOCTYPE html>
 <html>
-
-<head>
```

Comment:
This code shouldn't be missing, the <head> section of all pages is broken.
Should be :
```html
<head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <title>{% block title %}Loupelou{% endblock %}</title>

    {% block stylesheets %}
        # Insert stylesheets here
    {% endblock %}
</head>
```